drivers/driver-hazelcast4plus/src/main/java/com/hazelcast/simulator/worker/loadsupport/Main.java

```diff
@@ -0,0 +1,32 @@
+package com.hazelcast.simulator.worker.loadsupport;
+
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+
+public class Main {
+    public static void main(String[] args) throws Exception {
+        HazelcastInstance hz = Hazelcast.newHazelcastInstance();
+        // we ingest into a <Long,Long> IMap
+        IMapGeneratingIngestor.<Long, Long>configure()
+                // coordinated by instance hz
+                .withHazelcastInstance(hz)
+                // into map 'mymap'
+                .withMapName("mymap")
+                // ingest by sending tasks to all members, each task on every member to put 1000 generated entries
+                .withBatchSize(1000)
```

Comment:
I guess batch size, throttle and parallelism will have some reasonable defaults and setting it won't be mandatory, right? I can imagine that 99 % of the time, I will not change itl.

Reply:
Correct. Maybe with the exception of the parallelism. The more cores you have, the higher parallelism you want, otherwise data generation will be a bottleneck. The thing is the number of the cores is not known by the clients, so it should be set. What we can maybe do is the following:
1) send a task to each member that determines the `parallelism`, since the core count is already know there
2) then that tasks send the N actual load generator tasks

Maybe it could be used to determine the `throttle` as well.